src/general/MainMenu.cs

```diff
@@ -100,6 +100,17 @@ public void SetCurrentMenu(uint index, bool slide = true)
         }
     }
 
+    [RunOnKeyDown("ui_cancel")]
+    public void ReturnPressed()
```

Comment:
This should be private. And renamed to OnReturnToMainMenuPressed

Reply:
But I need to call it outside this class, if it is private, it cannot get called. Or I should add another public method and call that so the logic is more understandable.

The function name is as it is before, so this means I will rename it?

I will resolve this maybe two weeks later... I have an exam next weekend.

Reply:
I mean you added this new method? Why does this need to be called from outside this class?
Maybe you mean the RunOnkeyDownAttribute? I thought that it would work on private methods as well, but I guess maybe it doesn't? So that's why this needs to be public.

Reply:
Godot can access this Only if this method is the public method. I failed the first few times because this method was set to private.

Reply:
It's not really Godot, it's our input manager code. Which *could* be updated to also work with private methods, but for now I don't think it's worth the effort.